x-pack/plugins/observability_solution/observability_shared/public/components/feature_feedback_button/feature_feedback_button.tsx

```diff
@@ -23,18 +26,27 @@ const getDeploymentType = (isCloudEnv?: boolean, isServerlessEnv?: boolean): str
   return 'Self-Managed (you manage)';
 };
 
+const getMLJobType = (mlJobType: NodeType): string | undefined => {
+  if (mlJobType === 'pod') {
+    return 'Pod Anomalies';
+  }
+  return 'Host Anomalies';
+};
+
```

Comment:
If you want to write less code:
```suggestion
const getMLJobType = (mlJobType: NodeType) => mlJobType === 'pod' ? 'Pod Anomalies' : 'Host Anomalies';
```

And you've also missed to add i18n.

Reply:
I like the idea, thanks. No those are URL parameters we should not translate them